csrankings-t.csv

```diff
@@ -155,6 +155,7 @@ Tanzima Z. Islam,Texas State University,https://www.tanzimaislam.com,q56uneAAAAA
 Tanzima Zerin Islam,Texas State University,https://www.tanzimaislam.com,q56uneAAAAAJ
 Tao A. Xiang,University of Surrey,http://www.eecs.qmul.ac.uk/~txiang,MeS5d4gAAAAJ
 Tao Chen 0001,Loughborough University,https://www.lboro.ac.uk/departments/compsci/staff/academic-teaching/tao-chen,K4teyvoAAAAJ
+Tao Dai 0001,Shenzhen University,https://openreview.net/profile?id=~Tao_Dai3,MqJNdaAAAAAJ

```

Comment:
Replace openreview with actual web page.

Reply:
I post the official website homepage link and screenshot from College of Computer Science and Software Engineering in Shenzhen University. Please check if this homepage is eligible? Thank you so much. [https://csse.szu.edu.cn/en/pages/user/index?id=1204](url)
![Tao Dai](https://github.com/user-attachments/assets/5d15a33e-e818-4089-adf5-1873d8bb1c12)